### PR TITLE
Add chalk repository under automation

### PR DESCRIPTION
**Pull request by Kobzol**
Repo: https://github.com/rust-lang/chalk

Extracted from GH:
```
org = "rust-lang"
name = "chalk"
description = "An implementation and definition of the Rust trait system using a PROLOG-like logic solver"
bots = []

[access.teams]
security = "pull"
infra = "admin"
compiler = "write"
types = "write"
lang = "write"

[access.individuals]
rust-lang-owner = "admin"
oli-obk = "write"
pnkfelix = "write"
joshtriplett = "write"
aturon = "admin"
Aaron1011 = "write"
cjgillot = "write"
compiler-errors = "write"
wesleywiser = "write"
fabric-and-ink = "write"
rustbot = "maintain"
kennytm = "admin"
shepmaster = "admin"
tweerwag = "write"
scalexm = "admin"
tmandry = "write"
withoutboats = "admin"
rust-highfive = "write"
scottmcm = "write"
matthewjasper = "write"
estebank = "write"
davidtwco = "write"
badboy = "admin"
nikomatsakis = "write"
bors = "write"
BoxyUwU = "write"
jackh726 = "write"
michaelwoerister = "write"
Mark-Simulacrum = "admin"
rylev = "admin"
petrochenkov = "write"
aliemjay = "write"
jdno = "admin"
Kobzol = "admin"
nagisa = "write"
pietroalbini = "admin"
lcnr = "write"
spastorino = "write"
eddyb = "write"
```

### CODE REVIEW

**Comment by jackh726** [comment]
I think only types is fine, no compiler/lang.

**Comment by Kobzol** [comment]
Changed.

File: repos/rust-lang/chalk.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+org = "rust-lang"
+name = "chalk"
+description = "An implementation and definition of the Rust trait system using a PROLOG-like logic solver"
+bots = ["rustbot", "bors"]
+
+[access.teams]
+types = "write"
```
Comment (rylev) [inline repos/rust-lang/chalk.toml:7]: Is compiler and lang team access no longer needed?

Comment (Kobzol) [reply repos/rust-lang/chalk.toml:7]: See https://github.com/rust-lang/team/pull/1185#issuecomment-1884017701.

**Review by jackh726 — APPROVED** [review]

**Pull request merged**